pkg/stub/handler.go

```diff
@@ -203,22 +210,35 @@ func getPodReadyState(p *corev1.Pod) bool {
 	return ready
 }
 
-// updateDaemonSet creates a new DaemonSet object if it does not exist.
+// upsertDaemonSet creates a new DaemonSet object if it does not exist or
+// updates an existing one if changes need to be synchronized.
 //
 // Returns an error in the following conditions:
 //  - all k8s apierrors except IsNotFound
 //  - failure on daemonset creation
-func updateDaemonSet(oa *v1alpha1.OneAgent) error {
+func upsertDaemonSet(oa *v1alpha1.OneAgent) error {
 	ds := getDaemonSet(oa)
```

Comment:
ds --> daemonSet?
maybe, for better readability?

Reply:
The very short variable names are pretty common in go-lang.

I would leave it with `ds`.